app/assets/stylesheets/application.scss

```diff
@@ -696,3 +704,19 @@ margin: 0 auto;
       
     }
   }
+
+
+//   @media screen and (max-width: 1000px) {
+//     #account-group-1{
+//       justify-content: center;
+//       display: block;
+//     }
+
+//     #cq-body{
+//       margin: 20px auto;
+//     }
+
+//     .project-card{
+//       margin: 20px auto;
+//     }
+// }
```

Comment:
Need to keep the media query?

Reply:
no it was something I was messing around with I can delete it.